### ISSUE TITLE
Problem communicating with nordvpnd through socket

### ISSUE THREAD

**Issue by quoimec** [mmnaseri/nordvpn-reconnect#1]
Hello, thank you for taking the time to make this, this is such a pain with the current Nord package. 

Unfortunately through, I am running into some issues when running this script. I installed the service but came back to find that my vpn connection had been up for 10+ hours and had the DNS issue. I then tried manually running the script and got this response from the logs:

```
Initializing NordVPN reconnector with a maximum uptime of 4 hours
Desktop notifications are disabled.
Current uptime is 15 hours 30 minutes, which is greater than 4 hours
Attempting to reconnect.
-Attempting to kill process 2243 for service nordvpnd
./nordvpn-reconnect.sh: line 117: kill: (2243) - Operation not permitted
Failed to kill service; it may be that the service is running at a higher privilege than this script
Asking nordvpnd-killer.service to take care of this for us.
Whoops! /run/nordvpn/nordvpnd.sock not found
Whoops! /run/nordvpn/nordvpnd.sock not found
Checking again in 24 minutes
```

Any thoughts? I don't think I did anything strange when setting up Nord so I am a bit stumped. 

Any help much appreciated!

**Comment by mmnaseri** [comment]
Hi,

Did you ever add your user to the `nordvpn` group?

```
sudo usermod -aG nordvpn $USER
```

**Comment by quoimec** [comment]
Yep, my user account appears to be part of the NordVPN group

```
pi@raspberrypi:~ $ groups pi
pi : pi adm dialout cdrom sudo audio video plugdev games users input netdev spi i2c gpio docker nordvpn debian-transmission
```

**Comment by mmnaseri** [comment]
I'm not able to reproduce this. Can you actually verify that the socket exists?

```bash
stat /run/nordvpn/nordvpnd.sock
```

and can you check to see if the service is up and running?

```bash
pgrep nordvpnd
```

**Comment by mmnaseri** [comment]
Did you get this working? Is it possibly related to the NordVPN client issue discussed in #5?

**Comment by N72826** [comment]
same issue for me, we're both on a pi. I'm on ubuntu, when I attempted using nord on raspbian os, it was more unstable. I still receive the same message, nord reconnector is incapable of killing the process due to a permission issue despite adding the default user to the nordvpn group. after it is force killed by the sister script (nordvpnd-killer), nord then cannot start correctly without a reboot of the device. I attached a system log to my thread.
